src/pages/ProductList/List/PremiumCard.jsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import styled from "styled-components";
+import variables from "../../../styles/variables";
+
+const PremiumCard = ({ product, isNearBy }) => {
+  const {
+    name,
+    time_price,
+    stay_price,
+    distance,
+    thumbnail_url,
+    avg_rating,
+    count_rating,
+  } = product;
+
+  const priceToString = price => {
+    if (price !== null) {
+      return price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",") + "원";
+    } else {
+      return "가격정보없음";
+    }
+  };
+
+  const address = product.address.split(" ");
+
+  const ratingMessage = rating => {
+    let message;
+    if (rating < 3) {
+      message = "별로에요";
+    } else if (rating < 9) {
+      message = "추천해요";
+    } else {
+      message = "최고에요";
+    }
+    return message;
+  };
+
+  const rating = (rating, count) => {
+    if (rating !== null) {
+      return `${Math.round(rating * 10) / 10} ${ratingMessage(
+        avg_rating
+      )} (${count})`;
+    } else {
+      return "별점 정보 없음";
+    }
+  };
```

Comment:
```suggestion
  const rating = (rating, count) => {
    if(!rating){
      return "별점 정보 없음";
    }
    return `${Math.round(rating * 10) / 10} ${ratingMessage(
        rating
      )} (${count})`;
  };
```
좀 더 깔끔하게 작성할 수도 있겟네요!

Reply:
반영했습니다 감사합니다!